com.unity.netcode.gameobjects/Tests/Runtime/NetworkObject/NetworkObjectOnSpawnTests.cs

```diff
@@ -168,6 +168,23 @@ bool HasConditionBeenMet()
             Assert.False(s_GlobalTimeoutHelper.TimedOut, "Timed out while waiting for client side despawns! (2nd pass)");
         }
 
+        [Test]
+        public void DynamicallySpawnedNoSceneOriginException()
```

Comment:
with this additional test, are we confident enough to say we cover all expected cases now? or are there going to be some stuff left outside the scope like one LP mentioned on the Slack thread which lead to this fix? in short, I wonder how confident are we in testing SceneHandle/Origin stuff now? do we need other tests etc.?